Apps/Sandcastle/gallery/Japan Buildings.html

```diff
@@ -0,0 +1,286 @@
+  </head>
+  <body
+    class="sandcastle-loading"
+    data-sandcastle-bucket="bucket-requirejs.html"
+  >
+    <style>
+      @import url(../templates/bucket.css);
+    </style>
+    <div id="cesiumContainer" class="fullSize"></div>
+    <div id="loadingOverlay"><h1>Loading...</h1></div>
+    <div id="toolbar"></div>
+    <script id="cesium_sandcastle_script">
+      window.startup = async function (Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+
+        const viewer = new Cesium.Viewer("cesiumContainer", {
+          terrain: Cesium.Terrain.fromWorldTerrain(),
+          baseLayerPicker: false,
+          sceneModePicker: false,
+          animation: false,
+          timeline: false,
+          msaaSamples: 4,
+        });
+        const scene = viewer.scene;
+        scene.globe.depthTestAgainstTerrain = true;
+
+        try {
+          // Load the Japan Building data asset
+          const tileset = await Cesium.Cesium3DTileset.fromIonAssetId(2602291);
+          scene.primitives.add(tileset);
+          await viewer.zoomTo(tileset);
+        } catch (error) {
+          console.log(error);
+        }
+
+        // Define camera locations for points of interest
+        const hiroshima = {
+          destination: new Cesium.Cartesian3(
+            -3558117.165209301,
+            3887175.058311886,
+            3582090.381367681
+          ),
+          orientation: {
+            direction: new Cesium.Cartesian3(
+              0.915045649098936,
+              -0.16130516440787898,
+              0.3696919041586574
+            ),
+            up: new Cesium.Cartesian3(
+              -0.20924973638933655,
+              0.5937271886242537,
+              0.7769829942214522
+            ),
+          },
+        };
+
+        const tokyoTower = {
+          destination: new Cesium.Cartesian3(
+            -3959788.9678092706,
+            3353283.9088315447,
+            3697270.0292328526
+          ),
+          orientation: {
+            direction: new Cesium.Cartesian3(
+              0.1473261076519599,
+              -0.9210400676146971,
+              0.3605276852787276
+            ),
+            up: new Cesium.Cartesian3(
+              -0.6082716434343354,
+              0.20305763470537083,
+              0.7673155835649066
+            ),
+          },
+        };
+
+        const kyotoNijoCastle = {
+          destination: new Cesium.Cartesian3(
+            -3746418.0787567603,
+            3649244.7209161296,
+            3638967.47570257
+          ),
+          orientation: {
+            direction: new Cesium.Cartesian3(
+              0.9417381486076588,
+              -0.026110036454204615,
+              0.335331963065526
+            ),
+            up: new Cesium.Cartesian3(
+              -0.2518896785254185,
+              0.6059364940549604,
+              0.7545810460280222
+            ),
+          },
+          duration: 0,
+          easingFunction: Cesium.EasingFunction.LINEAR_NONE,
```

Comment:
Is this necessary with the duration of `0`?

Reply:
no need